daily_milestone.go

```diff
@@ -85,7 +86,53 @@ func getProjectID(baseURL string, Token string, Projectname string, Namespace st
 	return strconv.Itoa(project.ID)
 }
 
-// CreateMilestoneData is used to check the due date using the time package of python
+func getMilestones(baseURL string, token string, projectID string) []string {
+	project := gitLabAPI{}
+	list := []string{}
+	strurl := []string{"https://", baseURL, "/projects/", projectID, "/milestones"}
+
+	page := 1
+
+	for {
+		url := strings.Join(strurl, ",")
+		client := &http.Client{}
+		re := regexp.MustCompile("^[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9]$")
```

Comment:
Just as a sidenote. Most likely the better thing to do here is to parse it with a time library and check, if the format is valid, but for now that's alright like this.

Reply:
Will look into that also please take a look at deleting element from slice it is not as straight forward as Python and confuses me ;)